INSTALL.md

```diff
@@ -406,30 +333,12 @@ data represents a statistical natural language processing model of address parsi
 OpenStreetMap data. The API will also require about 4GB of memory (it used only a few hundred
 before), to store the needed data for queries.
 
-First, install libpostal following its [installation docs](https://github.com/openvenues/libpostal#installation).
-This will also download the training data, so be sure to have enough free disk space.
```

Comment:
No I meant you can keep a link to the installation docs here, to remind people where the directions can be found.